### PR TITLE
New users list and fix api news Kucher

### PR DESCRIPTION
**Pull request by MetalMari**

### CODE REVIEW

**Review by Viktorika-p — APPROVED** [review]

File: front-end/src/Components/Common/CategoriesDropDown.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class CategoriesDropDown extends React.Component {
         // Geting list of category from API
```
Comment (vvelizariy) [inline front-end/src/Components/Common/CategoriesDropDown.js]: // Get categories

File: front-end/src/Components/pages/userslistPage/RolesDropDown.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import React, { Component } from "react";
+import { Form } from "react-bootstrap";
+
+class RolesDropDown extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```
Comment (vvelizariy) [inline front-end/src/Components/pages/userslistPage/RolesDropDown.js]: state can be initialized without constructor

File: front-end/src/Components/Common/CategoriesDropDown.js (hunk 1 above)
Comment (MetalMari) [reply front-end/src/Components/Common/CategoriesDropDown.js]: Changed

File: front-end/src/Components/pages/userslistPage/RolesDropDown.js (hunk 2 above)
Comment (MetalMari) [reply front-end/src/Components/pages/userslistPage/RolesDropDown.js]: Changed

**Review by OleksandrHavron — APPROVED** [review]

**Review by Maksson — APPROVED** [review]

**Review by s0fko — APPROVED** [review]

**Pull request merged**